### PR TITLE
Reduce clutter in log from course update jobs

### LINKED ISSUE #2: Reduce clutter in log from course update jobs
The job logs for updating courses have a lot of clutter in them that is unnecessary now that we have an updates table that tracks the success/failure of the course updates.

Running the job to update the entire database for a large range of quarters could cause the job log to get unmanageably long unless we trim this down a bit.

We do want some output to track the progress of updating courses, but what we have now is a bit too much.  Make some choices that reduce the amount of log output, while still retaining the ability to track the progress of the job.

### PR DESCRIPTION
**Pull request by eugeneLim628**
Closes #2 
This PR reduced unnecessary print statements from the code 
Got rid of "Updating MongoDB Collection", "Found # sections" "Error saving section" and "Data is not stale" statements that were being printed by the logs and changed the tests to also account for this. 
As a result, there is less text in the logs and therefore easier to read.

### CODE REVIEW

**Review by pconrad — CHANGES_REQUESTED** [review]
Please remove commented out code througout

https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

File: src/test/java/edu/ucsb/cs156/courses/jobs/UpdateCourseDataJobTests.java [hunk 1]
```diff
@@ -382,9 +361,9 @@ void test_if_stale_and_is_not_stale() throws Exception {
 
     // Assert
 
-    String expected = "Data is not stale for [MATH 20211]";
+    /*String expected = "Data is not stale for [MATH 20211]";
 
     assertEquals(expected, jobStarted.getLog());
-    verify(isStaleService, times(1)).isStale(eq("MATH"), eq("20211"));
+    verify(isStaleService, times(1)).isStale(eq("MATH"), eq("20211"));*/
```
Comment (ChristopherChau) [inline src/test/java/edu/ucsb/cs156/courses/jobs/UpdateCourseDataJobTests.java:367]: I think you need to remove the commented code based on the professor's requested change

Comment (pconrad) [reply src/test/java/edu/ucsb/cs156/courses/jobs/UpdateCourseDataJobTests.java:367]: Absolutely correct.  This must be removed before this is merged for points.

**Review by pconrad — CHANGES_REQUESTED** [review]
https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html#why-should-commented-out-code-be-removed

**Comment by pconrad** [comment]
> I reduced unnecessary print statements from my code

It isn't "your code". It's "the code" that we all share.

> Got rid of "Updating MangoDB Collection", "Found # sections" "Error saving section" and "Data is not stale" from logs and updated tests to reflect the changes 

And why is that helpful? The PR description should state that. (For example: reducing clutter in the logs make them more compact and easier to read.

Also: Mango?   Check your spelling.

File: src/test/java/edu/ucsb/cs156/courses/jobs/UpdateCourseDataJobTests.java (hunk 1 above)
Comment (eugeneLim628) [reply src/test/java/edu/ucsb/cs156/courses/jobs/UpdateCourseDataJobTests.java]: I fixed it !

**Review by ChristopherChau — APPROVED** [review]
LGTM, nice changes

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**